apps/dashr-multipage-report/app.R

```diff
@@ -0,0 +1,1916 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(stringi)
+library(generator)
+library(plotly)
+library(random)
+library(data.table)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-multipage-report")
+
+supplyDemand <- fread("/Data/supplyDemand.csv")
```

Comment:
I think you have to supply relative paths here, so either `Data/supplyDemand.csv` or `./Data/supplyDemand.csv`